### PR TITLE
EPMLABSBRN-296 - BackEnd: update task / exercise if in new file came object with same constrain

### PR DESCRIPTION
**Pull request by RiveriRiv**
[EPMLABSBRN-296](https://jira.epam.com/jira/browse/EPMLABSBRN-296)

**Description**:

Added updating exercises while loading 2 series

### CODE REVIEW

File: src/main/kotlin/com/epam/brn/csv/converter/impl/secondSeries/Exercise2SeriesConverter.kt [hunk 1]
```diff
@@ -34,19 +38,39 @@ class Exercise2SeriesConverter : Converter<Map<String, Any>, Exercise> {
     @Autowired
     lateinit var seriesService: SeriesService
 
+    @Autowired
+    lateinit var exerciseService: ExerciseService
```
Comment (ElenaSpb) [inline src/main/kotlin/com/epam/brn/csv/converter/impl/secondSeries/Exercise2SeriesConverter.kt]: could you please move this dependancy injections into constructor

**Comment by ElenaSpb** [comment]
what about tasks entities?

File: src/main/kotlin/com/epam/brn/csv/converter/impl/secondSeries/Exercise2SeriesConverter.kt (hunk 1 above)
Comment (RiveriRiv) [reply src/main/kotlin/com/epam/brn/csv/converter/impl/secondSeries/Exercise2SeriesConverter.kt]: done

File: src/main/kotlin/com/epam/brn/csv/converter/impl/secondSeries/Exercise2SeriesConverter.kt [hunk 2]
```diff
@@ -28,25 +35,36 @@ class Exercise2SeriesConverter : Converter<Map<String, Any>, Exercise> {
...
+        val level = source[LEVEL].toString().toInt()
+
+        return try {
+            val exercise = exerciseService.findExerciseByNameAndLevel(name, level)
+
+            log.debug("exercise with name {$name} and level {$level} is already persisted. Entity Will be updated")
+
+            exercise
+        } catch (e: EntityNotFoundException) {
+            val exercise = Exercise()
```
Comment (odanc) [inline src/main/kotlin/com/epam/brn/csv/converter/impl/secondSeries/Exercise2SeriesConverter.kt]: It's data class, so you can
`Exercise(name = name, level = level)`

Comment (RiveriRiv) [reply src/main/kotlin/com/epam/brn/csv/converter/impl/secondSeries/Exercise2SeriesConverter.kt]: done

**Pull request merged**